packages/app-desktop/gui/TagList.tsx

```diff
@@ -13,6 +14,7 @@ interface Props {
 }
 
 function TagList(props: Props) {
+	const collatorLocale = currentLocale().slice(0, 2);
```

Comment:
I think you misunderstood Roman's comment - there are functions in lib/locale to parse these strings. For example the `languageName` function and you should use this instead of string manipulation